frontend/src/app/patients/UploadPatients.test.tsx

```diff
@@ -383,20 +383,35 @@ describe("Upload Patient", () => {
   });
   it("should show loading message when file is being processed", async () => {
     const { user } = renderUploadPatients();
+    const errorResponseBody = {
+      status: "FAILURE",
+      errors: [
+        {
+          indices: [],
+          message: "File is missing headers and other required data",
+        },
+      ],
+    };
+    let mockResponse = new Response(JSON.stringify(errorResponseBody), {
+      status: 200,
+    });
     jest
       .spyOn(FileUploadService, "uploadPatients")
       .mockImplementation(async () => {
-        await new Promise((resolve) => setTimeout(resolve, 500));
-        return Promise.resolve(new Response());
```

Comment:
this was causing a 
![image](https://github.com/CDCgov/prime-simplereport/assets/4952042/553a3b70-909b-4f56-87df-6e862122aa17)

error some of the time, only if the timeout resolved while other tests are still running and it would incorrectly cause the other test to fail.
tests are not isolated!